backend/src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -3388,4 +3388,52 @@ databaseChangeLog:
                    (gen_random_uuid(), 'Flu B', 'LP14240-3');
       rollback:
         sql: |
-          TRUNCATE TABLE ${database.defaultSchemaName}.supported_disease CASCADE;
\ No newline at end of file
+          TRUNCATE TABLE ${database.defaultSchemaName}.supported_disease CASCADE;
+  - changeSet:
+      id: create-device-disease-join-table
+      author: emma@skylight.digital
+      comment: Add a new table that tracks which disease a given testing device will support.
+      changes:
+        - tagDatabase:
+            tag: create-device-disease-join-table
+        - createTable:
+            tableName: device_supported_disease
+            remarks: A usable combination of device type and supported diseases for that device.
+            columns:
+              - column:
+                  name: internal_id
+                  type: uuid
+                  remarks: The internal database identifier for this entity.
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column: *created_at_column
+              - column: *created_by_column
+              - column: *updated_at_column
+              - column: *updated_by_column
+              - column: *soft_delete_column
```

Comment:
why do we need `created_at_column, created_by_column, updated_at_column, updated_by_column, soft_delete_column` on a join table
also soft_deletion is not possible on a join table
for example https://github.com/CDCgov/prime-simplereport/blob/7d76bd474643d5af58ee00e2784929545417e811/backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceType.java#L33
here there was no way for us to tell the `@JoinTable` annotation to ignore certain entries, thus soft deletion on `device_specimen_types` was not possible.
a pure join table should only have two columns, we tried it before and now we're paying the price of treating `device_specimen_types` as a dataobject and not as a join-table by having to do some complicated fixes.

Reply:
Discussed offline - I have no strong feelings on making this auditable, I only did it because the join table example I was using as an example was auditable. But more than happy to remove! Will push a commit shortly

Reply:
Audit columns removed; please re-review when you get the chance!